src/views/GovernanceReviewTeam/RequestOverview.test.tsx

```diff
@@ -26,35 +33,436 @@ jest.mock('@okta/okta-react', () => ({
   }
 }));
 
-describe('The GRT Review page', () => {
-  it('renders the Summary component', async () => {
-    const mockStore = configureMockStore();
-    const store = mockStore({
-      auth: {
-        groups: ['EASI_D_GOVTEAM'],
-        isUserSet: true
-      },
-      systemIntake: {
+window.matchMedia = (): any => ({
+  addListener: () => {},
+  removeListener: () => {}
+});
+
+const waitForPageLoad = async () => {
+  await waitForElementToBeRemoved(() => screen.getByTestId('page-loading'));
+};
+
+describe('Governance Review Team', () => {
+  const intakeQuery = {
```

Comment:
This test data is filled out more thoroughly than needed. I added this before I scoped out what tests would be in this PR.

I'd like to keep this here for now so I don't lose it. Once I am able to use this query in other tests, I'll strip this down to the bare minimum so it's not too chunky.